components/phpunit_bridge.rst

```diff
@@ -565,6 +565,76 @@ conditions::
         ],
     ]);
 
+Class Existence Based Tests
+---------------------------
+
+Tests that behave differently depending on existing classes, for example Composer's
+development dependencies, are often hard to test for the alternate case. For that
+reason, this component also provides mocks for these PHP functions:
+
+* :phpfunction:`class_exists`
+* :phpfunction:`interface_exists`
+* :phpfunction:`trait_exists`
+
+Use Case
+~~~~~~~~
+
+Consider the following example that relies on the ``Vendor\DependencyClass`` to
+toggle a behavior::
+
+    use Vendor\DependencyClass;
+
+    class MyClass
+    {
+        public function hello(): string
+        {
+            if (class_exists(DependencyClass::class)) {
+                return 'The dependency bahavior.';
+            }
+
+            return 'The default behavior.';
+        }
+    }
+
+A regular test case for ``MyClass`` (assuming the development dependencies
+are installed during tests) looks like::
+
+    use PHPUnit\Framework\TestCase;
+
+    class MyTest extends TestCase
+    {
+        public function testHello()
+        {
+            $class = new MyClass();
+            $result = $class->hello(); // "The dependency bahavior."
```

Comment:
Could you please use an assertion here? Otherwise the code example is not valid when someone copies it, thank you

Reply:
Hmm I saw here 
https://github.com/symfony/symfony-docs/pull/11904#pullrequestreview-258628191
that we didn’t use an assertion here. But I would prefer using one in the other examples too.

WDYT @javiereguiluz